pkg/devfile/adapters/common/utils.go

```diff
@@ -146,3 +147,19 @@ func IsPortPresent(endpoints []common.DockerimageEndpoint, port int) bool {
 
 	return false
 }
+
+// IsRestartRequired returns if restart is required for devrun command
+func IsRestartRequired(command common.DevfileCommand) bool {
+	var restart = true
+	var err error
+	rs, ok := command.Attributes["restart"]
```

Comment:
If here the `ok` is false i.e. restart is not present then wouldn’t restart be returned as `true` due to line 153?

Reply:
yes, if restart is not present, it should be true. Which is the default current behavior.
```
restart not present         true
restart: true               true
restart: false              false
```

Reply:
Gotcha